examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -78,7 +85,7 @@
 
 
 # locations relative to project root
-root = Path(timemachine.__file__).parent
```

Comment:
I think this actually should be `root = Path(timemachine.__file__).parent.parent`

Reply:
Yeah this tripped me up too, it looks like it was importing timemachine.timemachine as opposed to timemachine. I thought pinning it relative to the actual file itself might be more robust.

Reply:
My argument against pinning it relative to the file is that this will likely be moved out of the hif2a examples dir as it is no longer specific to hif2a. Also someone uses it as an example the path will be correct even if they have the script somewhere else

Reply:
Okay I can replace it with `root = Path(timemachine.__file__).parent.parent`